manifest_staging/charts/gatekeeper/templates/gatekeeper-audit-deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         {{- toYaml .Values.auditPodAnnotations | trim | nindent 8 }}
         {{- end }}
       labels:
-{{- include "gatekeeper.podLabels" . }}
```

Comment:
Instead of doing it here, you can add your changes here: https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/main.go#L139

e.g.
> if kind == DeploymentKind {
				obj = strings.Replace(obj, "      labels:", "      labels:\n        {{- include \"gatekeeper.podLabels\" . | nindent 8 }}", 1)
			}

then run "make manifests" for these files

Reply:
Thank you again, I managed to update the manifests with the make step. PR updated 🙏